network/Message.cpp

```diff
@@ -404,15 +404,13 @@ Message TurnProgressMessage(Message::TurnProgressPhase phase_id) {
     return Message(Message::TURN_PROGRESS, os.str());
 }
 
-Message PlayerStatusMessage(int about_player_id,
-                            Message::PlayerStatus player_status,
+Message PlayerStatusMessage(Message::PlayerStatus player_status,
```

Comment:
Swapping the order of these causes a lot of changes elsewhere. Is it necessary or beneficial?

Reply:
It was player ID, now there empire ID and both have type int. So I decide to swap order to don't send player ID as empire ID somewhere.